src/parser/formatting-evaluator-module.ts

```diff
@@ -56,15 +56,15 @@ export class FormattingEvaluatorModule implements Module {
         const comment = comments[i];
         // Count with html elements if any, otherwise just treat it as plain text
         const { formatting } = this._getFormattingScore(comment);
-        const multiplierFactor = this._multipliers?.[comment.type] ?? { wordValue: 0, formattingMultiplier: 0 };
+        const multiplierFactor = this._multipliers?.[comment.type] ?? { multiplier: 0 };
         const formattingTotal = formatting
           ? Object.values(formatting).reduce((acc, curr) => {
               let sum = new Decimal(0);
               for (const symbol of Object.keys(curr.symbols)) {
                 sum = sum.add(
                   new Decimal(curr.symbols[symbol].count)
                     .mul(curr.symbols[symbol].multiplier)
-                    .mul(multiplierFactor.formattingMultiplier)
+                    .mul(multiplierFactor.multiplier)
                     .mul(curr.score)
                 );
               }
```

Comment:
can you help me grasp how this calculation work?

At first for every tag (`<a>`), the number of regex matches is counted (each regex has its own multiplier) and has a so called score which is defined in the config.

Then for the calculation, for every regex (symbol) the count is multiplied by the regex multiplier which makes sense, then that is multiplied by a "general" multiplier and multiplied by a `score` which is a multiplier for the tag I assume.

I had to read the code a couple of times and it's hard to keep track of all different names for same things, like symbols = regex, score = multiplier for a tag and variables names are confusing such as `wordCount` for regex count

Reply:
At least in the config we renamed things accordingly. For example I'm pretty sure we renamed symbol to regex, so we should do the same in the code here. 

Also score should just sum per instance. For example if we want to award $1 for every paragraph, then score for p should be 1. 

Score is not supposed to be a multiplier.

Reply:
I will rename the variables to match the configuration. The `score` concept does not exist (yet) in v2 and can be added in a separate PR. Only multipliers exist for now.

Reply:
> Also `score` should just sum per instance. For example if we want to award $1 for every paragraph, then `score` for `p` should be `1`.
> 
> `score` is not supposed to be a multiplier.

that was also my understanding but currently in the code it's a multiplier